firmware/application/ui/ui_receiver.hpp

```diff
@@ -35,6 +35,8 @@
 #include <algorithm>
 #include <functional>
 
+#define MAX_UFREQ 7200000000  // maximum usable frequency
```

Comment:
Isn't this defined elsewhere? Would be good to have a single source of truth.

Reply:
So this is used for the freq control.

```
FrequencyField::FrequencyField(
    const Point parent_pos)
    : Widget{{parent_pos, {8 * 10, 16}}},
      length_{11},
      range_{rf::tuning_range} {. <----
```

and it's defined as
`constexpr FrequencyRange tuning_range{path::band_low.minimum, path::band_high.maximum};`

Ideally more things would use that -- can be a change for another PR. I'll add a note in my cleanup list.

Reply:
In another PR then.